### PR TITLE
AF-2125: Spliting ace and core editors from base widgets

### PR DESCRIPTION
**Pull request by jesuino**
Part of an ensemble

https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1066
https://github.com/kiegroup/kie-wb-common/pull/2917
kiegroup/drools-wb#1235
kiegroup/jbpm-wb#1393
kiegroup/optaplanner-wb#347
kiegroup/kie-wb-distributions#964

### CODE REVIEW

**Comment by jesuino** [comment]
Jenkins execute full downstream build

**Comment by jesuino** [comment]
jenkins retest this

**Comment by domhanak** [comment]
jenkins retest this

**Comment by jesuino** [comment]
jenkins retest this

**Comment by jesuino** [comment]
Jenkins execute full downstream build

**Comment by domhanak** [comment]
The builds don't work properly

**Comment by domhanak** [comment]
jenkins retest this

**Comment by jesuino** [comment]
Jenkins execute full downstream build

**Comment by domhanak** [comment]
jenkins retest this

**Comment by jesuino** [comment]
Jenkins execute full downstream build

**Comment by domhanak** [comment]
@jesuino So, the tests in `droolsjbpm-integration` are failing for some reason. Could you look into it? I will review in the meantime.

File: uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-editors/src/main/java/org/uberfire/ext/widgets/core/client/resources/i18n/EditorsConstants.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+package org.uberfire.ext.widgets.core.client.resources.i18n;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.i18n.client.Messages;
+
+/**
+ * Editors Widgets I18N constants
+ */
+public interface EditorsConstants
```
Comment (domhanak) [inline uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-editors/src/main/java/org/uberfire/ext/widgets/core/client/resources/i18n/EditorsConstants.java]: i would rename this to `EditorConstants`  but just a nitpick :)

File: uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-editors/src/main/resources/org/uberfire/ext/widgets/core/client/resources/i18n/EditorsConstants_de.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+MetaFileEditor=Meta Datei Editor
```
Comment (domhanak) [inline uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-editors/src/main/resources/org/uberfire/ext/widgets/core/client/resources/i18n/EditorsConstants_de.properties]: Should this be `Metadatei-Editor` ?

File: uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-editors/src/main/java/org/uberfire/ext/widgets/core/client/resources/i18n/EditorsConstants.java (hunk 1 above)
Comment (domhanak) [inline uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-editors/src/main/java/org/uberfire/ext/widgets/core/client/resources/i18n/EditorsConstants.java]: I think this should be `enum`

**Comment by domhanak** [comment]
Roses are red,
violets are blue,
Jenkins execute full downstream build!

File: uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-editors/src/main/resources/org/uberfire/ext/widgets/core/client/resources/i18n/EditorsConstants_de.properties (hunk 2 above)
Comment (jesuino) [reply uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-editors/src/main/resources/org/uberfire/ext/widgets/core/client/resources/i18n/EditorsConstants_de.properties]: This came from previous files, I just moved it, let me know if you think that I should modify it

File: uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-editors/src/main/java/org/uberfire/ext/widgets/core/client/resources/i18n/EditorsConstants.java (hunk 1 above)
Comment (jesuino) [reply uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-editors/src/main/java/org/uberfire/ext/widgets/core/client/resources/i18n/EditorsConstants.java]: This also came from other modules, I just copied to this separated package

**Comment by domhanak** [comment]
@jesuino I see that there are constant failures here. Could you please check?

**Comment by jesuino** [comment]
The failures looks unrelated, let me try again because this is a huge change - all webapps are impacted

**Comment by jesuino** [comment]
Jenkins execute full downstream build

**Comment by domhanak** [comment]
jenkins execute compile downstream build

**Comment by ederign** [comment]
@tiagobento could you please take a look at this?

**Comment by jesuino** [comment]
Jenkins execute full downstream build

**Comment by jesuino** [comment]
Jenkins execute full downstream build
